presto-memsql/src/test/java/io/prestosql/plugin/memsql/TestMemSqlIntegrationSmokeTest.java

```diff
@@ -193,8 +207,7 @@ public void testLimitPushdown()
         assertThat(query("SELECT name FROM nation WHERE regionkey = 3 LIMIT 5")).isCorrectlyPushedDown();
 
         // with filter over varchar column
-        // TODO (https://github.com/prestosql/presto/issues/5263) should be `.isNotFullyPushedDown(FilterNode.class)`
-        assertThat(query("SELECT name FROM nation WHERE name < 'EEE' LIMIT 5")).isCorrectlyPushedDown();
+        assertThat(query("SELECT name FROM nation WHERE name < 'EEE' LIMIT 5")).isNotFullyPushedDown(FilterNode.class);
```

Comment:
Fixes #5263

Reply:
is the assumption about "memsql being case insensitive by default", correct?

can we have a ATQ / AbstractTestIntegrationSmokeTest entry validating connectors' correctnes across the board

Reply:
@findepi Yes. It's case insensitive by default.